src/CachePool.php

```diff
@@ -46,16 +51,17 @@ public function __construct(Cache $cache)
     /**
      * {@inheritdoc}
      */
-    public function getItem($key)
+    public function getItem($key, array $tags = array())
```

Comment:
No, it does not since the second argument is optional. 
If CacheItemPoolInterface were an abstract class it would.